markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -1,9 +1,87 @@
 #include <ctre/phoenix/motorcontrol/SupplyCurrentLimitConfiguration.h>
 #include <unistd.h>
 #include <string>
+#include <std_msgs/Float64.h>
 
 #include "markhor_hw_interface_flippers.hpp"
 
+static ros::Publisher fr_target_pub;
+static ros::Publisher fl_target_pub;
+static ros::Publisher rr_target_pub;
+static ros::Publisher rl_target_pub;
+
+static ros::Publisher fr_motor_current_pub;
+static ros::Publisher fl_motor_current_pub;
+static ros::Publisher rr_motor_current_pub;
+static ros::Publisher rl_motor_current_pub;
+
+bool MarkhorHWInterfaceFlippers::publishFRTarget(float target)
+{
+  std_msgs::Float64 msg;
+  msg.data = target;
+  ROS_INFO("publishFRTarget: %f \n", target);
+  fr_target_pub.publish(msg);
+  return true;
+}
+
+bool MarkhorHWInterfaceFlippers::publishFLTarget(float target)
+{
+  std_msgs::Float64 msg;
+  msg.data = target;
+  ROS_INFO("publishFLTarget: %f \n", target);
+  fl_target_pub.publish(msg);
+  return true;
+}
+
+bool MarkhorHWInterfaceFlippers::publishRRTarget(float target)
+{
+  std_msgs::Float64 msg;
+  msg.data = target;
+  ROS_INFO("publishRRTarget: %f \n", target);
+  rr_target_pub.publish(msg);
+  return true;
+}
+
+bool MarkhorHWInterfaceFlippers::MarkhorHWInterfaceFlippers::publishRLTarget(float target)
```

Comment:
Does this works ? It shouldn't be `MarkhorHWInterfaceFlippers::MarkhorHWInterfaceFlippers` but just `MarkhorHWInterfaceFlippers::`